compiler/rustc_middle/src/middle/codegen_fn_attrs.rs

```diff
@@ -120,9 +120,7 @@ bitflags::bitflags! {
         /// #[ffi_const]: applies clang's `const` attribute to a foreign function
         /// declaration.
         const FFI_CONST                 = 1 << 12;
-        /// #[cmse_nonsecure_entry]: with a TrustZone-M extension, declare a
-        /// function as an entry function from Non-Secure code.
-        const CMSE_NONSECURE_ENTRY      = 1 << 13;
+        // (Bit 13 was used for `#[cmse_nonsecure_entry]`, but is now unused.)
```

Comment:
Not sure if this comment is needed. I guess bit 7 maybe also was used previously but no more? There's no comment there.

Reply:
we are just following bit 14 here

```
        // (Bit 13 was used for `#[cmse_nonsecure_entry]`, but is now unused.)
        // (Bit 14 was used for `#[coverage(off)]`, but is now unused.)
```

I'm not sure what happened to bit 7, but it seems useful to document what bits were used for (e.g. if we ever run out of bits in this 32-bit integer, and might want to re-use one of the unused ones).